### PR TITLE
add a docker-compose set for running locally

### PR DESCRIPTION
**Pull request by aussielunix**
I love this repo as a simple app to show off the basics of Hotwire.
This PR adds a simple way to spin the app up and down for demos using Docker.

### CODE REVIEW

**Comment by khash** [comment]
Thank you for the PR. This however changes the app in a way that it can't run locally without Docker anymore. Can it be an option instead of the only way forward?

**Comment by aussielunix** [comment]
@khash  What I have add is already optional.
Nothing stopping anyone from installing ruby, bundler etc and running rails on their laptop directly.
This is just about making it extremely easy to see a working demo without having to have a Ruby environment on your laptop.

File: Gemfile.lock [hunk 1]
```diff
@@ -218,7 +216,7 @@ GEM
     zeitwerk (2.4.2)
 
 PLATFORMS
-  ruby
+  x86_64-linux-musl
```
Comment (khash) [inline Gemfile.lock:219]: Does this limit the non-docker environments?

File: config/cable.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 development:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: redis://tickerizer_redis:6379/1
```
Comment (khash) [inline config/cable.yml:3]: This limits the ability to run the app outside of Compose.

File: Gemfile.lock (hunk 1 above)
Comment (khash) [inline Gemfile.lock:219]: Does this limit the non-docker environments?

File: config/webpacker.yml [hunk 3]
```diff
@@ -58,10 +58,10 @@ development:
   # Reference: https://webpack.js.org/configuration/dev-server/
   dev_server:
     https: false
-    host: localhost
+    host: tickerizer_webpacker
```
Comment (khash) [inline config/webpacker.yml:61]: Compose only?

**Review by khash — COMMENTED** [review]
Thanks. I added some comments to clarify the point above.

**Comment by aussielunix** [comment]
🤦  Thanks - I'll look into this and tidyup the PR.

**Pull request closed without merging**